inspirehep/modules/records/api.py

```diff
@@ -30,6 +30,7 @@
 from elasticsearch.exceptions import NotFoundError
 from flask import current_app
 
+from inspire_schemas.api import validate as inspire_schemas_validate
```

Comment:
I don't think we need the `[...] as inspire_schemas_validate`, as the `validate` method inside `InspireRecord` does not shadow the outer one.